clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -3796,6 +3796,14 @@ public DescribeShareGroupsResult describeShareGroups(final Collection<String> gr
                 .collect(Collectors.toMap(entry -> entry.getKey().idValue, Map.Entry::getValue)));
     }
 
+    // To do in a follow-up PR
+    @Override
+    public ListShareGroupOffsetsResult listShareGroupOffsets(final Map<String, ListShareGroupOffsetsSpec> groupSpecs,
+                                                             final ListShareGroupOffsetsOptions options) {
+        // To-do
+        throw new InvalidRequestException("The method is not yet implemented");
```

Comment:
nit: UnsupportedOperationException should be there, but nevermind for current PR.